www_src/lib/api-fake-data.js

```diff
@@ -0,0 +1,58 @@
+module.exports = {
```

Comment:
Mind if we call this `mock.js` or `mock.json` and keep it isolated to the project (map) page?